.gitignore

```diff
@@ -3,3 +3,4 @@ _site
 node_modules/
 .sass-cache
 .jekyll-metadata
+.bundle/
```

Comment:
Hi Carlos,

I am very confused as the "asf-site" branch seems to have removed the ability to build locally (Gemfile removed and all .md files removed) and now see all the generated .html files (yet these changes do not appear on the PR).